### ISSUE TITLE
Mouse hard bugging after ucr installation, how to unistall ?

### ISSUE THREAD

**Issue by JustRoll321** [evilC/UCR#229]
I've used UCR to remap a special joystick witch wasn't working with cemu (an emulator for wii u games), to make vjoy think than input with that joystick were keyboards inputs. But it doesn't work properly, so i've supressed the files, thinking than will be enough to unistall the files.
Sadly, my mouse just stop working when i do some action, i don't why them, like openning a new tab in a seperate window or press F1 when playing zelda breath of the wild on cemu emulator (i don't use vjoy anymore for that, just a regular xbox360 controller that just work fine), for the rest of the...even when i shut down my computer.
But if i reboot my pc, it comes back. The mouse still be detected, and when it stops working, changing mice is not working too. When it's bugging, all my usb ports just refuse to make any mice work, with is a pain. I've tried to use a command like /install/uninstall.bat in the admin cmd prompt (can't remember what it was exactly and where i find it, on a thread on github for someone who wanna unistall, the guy haven't answered after your reply, so don't know if the command is good or not) but it didn't work. I just want a proper way to uninstal it, to get my mouse back at the job, fairly regular and not dropping me at any time she want for whatever reason. 
Thanks in advance

**Comment by JustRoll321** [comment]
Really need help on that, any idea ?

**Comment by Priyamkashyap** [comment]
I also want to know how to remove it properly and remove its data

**Comment by evilC** [comment]
The app is not "Installed" and does nothing when not running

**Comment by evilC** [comment]
Are you maybe talking about the [other version of UCR](https://github.com/Snoothy/UCR)? If so, you need to uninstall Interception

**Comment by Priyamkashyap** [comment]
can i have your Number I want to send video to you of what happening to My UCR. I could not bind control with keyboard it only detecting mouse clicks and scroll. I want send a video to you

**Comment by evilC** [comment]
No. As I said, I DO NOT THINK YOU ARE EVEN USING THIS SOFTWARE, I think you are using a COMPLETELY DIFFERENT piece of software which is also called UCR.
I don't see any way that this piece of software could cause this issue as it's just an AHK script that has no effect whilst running. If you are indeed using the other UCR, then in order to get keyboard working, you WOULD have to have installed something (Interception) which COULD cause this issue.
This version of UCR is old, unsupported software. The new version of UCR is supported and has a discord channel

**Comment by mvsreehari** [comment]
> No. As I said, I DO NOT THINK YOU ARE EVEN USING THIS SOFTWARE, I think you are using a COMPLETELY DIFFERENT piece of software which is also called UCR. I don't see any way that this piece of software could cause this issue as it's just an AHK script that has no effect whilst running. If you are indeed using the other UCR, then in order to get keyboard working, you WOULD have to have installed something (Interception) which COULD cause this issue. This version of UCR is old, unsupported software. The new version of UCR is supported and has a discord channel

Discord link?
